packages/transitive-overlay/src/index.tsx

```diff
@@ -16,6 +16,8 @@ import { getRouteLayerLayout, patternToRouteFeature } from "./route-layers";
 import { drawArc, getFromToAnchors, itineraryToTransitive } from "./util";
 
 export { itineraryToTransitive };
+// import routeArrow from "./images/route_arrow.png";
+
```

Comment:
Remove this commented code.

Reply:
I'm still experimenting trying to figure out how to satisfy the CI. I'm bouncing back versions that have the arrows and don't to try and pinpoint what whether this is actually a webpack issue or a jest configuration issue. There's going to be some oddities for a while, but I'll get them all smoothed out for when it's all straightened out and flying right.

Reply:
From running `yarn unit -u` there are snapshot changes that need to go in. Can you try checking in the updated snapshots?